### ISSUE TITLE
Optional modules. Parking functionality.

### ISSUE THREAD

**Issue by javieralmancevo** [mapsme/omim#953]
Is it posible to create modules that can be optionally installed? If not are you open to the idea? I like the clean simple interface of the app and allowing external modules would allow for people to get the functionality they want without bothering others.

I specifically ask because I use the parking module in Osmand but would like to switch to your app. In reality it is just a quick access bookmark to mark the position where you last parked, but it is very useful.

**Comment by deathbaba** [comment]
Modules are good in general, but in our case it can be complicated: I don’t think Apple App store rules allow it.

**Comment by javieralmancevo** [comment]
I understand. Any chance of getting the parking functionality as default then? I was thinking it could work as part of the bookmark menu.

**Comment by deathbaba** [comment]
In general, you can relatively easy put a bookmark at your location now: select blue arrow and press a star in the bottom toolbar. Does it work in your case?

**Comment by javieralmancevo** [comment]
Yes, but you have to name it and more importantly, its quite hard to delete. Its not ideal for something you are going to be setting and removing sometimes several times a day. Maybe not a parking specific bookmark, but just a general quick bookmark could work. A special bookmark that you can see in the main bookmark screen with a delete sign next to it, so it can be easily removed. That way it can be used for parking or for whatever people chose to.

**Comment by deathbaba** [comment]
We're going to release track recording which can show you where have you left your car.

**Comment by javieralmancevo** [comment]
Ill be looking into that.
